bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/helpers/BookieShellCommand.java

```diff
@@ -43,6 +43,10 @@ public BookieShellCommand(String shellCmdName,
 
     @Override
     public int runCmd(String[] args) throws Exception {
+        if (args.length <= 0) {
```

Comment:
@murong00 I don't think this change is correct. there are commands that don't have any args.

Reply:
@sijie I find that only cookie related commands which extend `CookieCommand` using `BookieShellCommand`, so I think the other subcommands are not affected which extend `MyCommand`. Maybe the title of the issue and pr should modify, or any advice? Thanks.